opsdroid/web.py

```diff
@@ -13,8 +13,21 @@
 
 
 class Web:
-    """Create class for opsdroid Web server."""
+    """A Web class
 
+    This class will create opsdroid Web server
+
+    Args:
+        opsdroid: opsroid instance
+
+    Attributes:
+        opsdroid: opsroid instance
+        config: web config of opsdroid instance from configuration yaml files
+        web_app: web.Application instance containing web server configurations
+        runner: AppRunner of web_app
+        site: this contains self.runner, host, port, ssl_context
```

Comment:
This is an `aiohttp.web.TCPSite` object. I see you've listed the arguments that were used to initialise it, but it may be more helpful to point people to what it is.